.travis.yml

```diff
@@ -22,6 +22,7 @@ before_script:
   - ./scripts/sauce/sauce_connect_block.sh
 
 script:
+  - node_modules/.bin/tslint -c tslint.json 'lib/**/*.ts' 'test/**/*.ts'
```

Comment:
isn't this covered by format:enforce ? 
if not what about creating a gulp task ?